examples/etcd/etcd-in-cluster/ocp/blueprint-v2/pv-etcd-backup.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-volume-etcd
```

Comment:
pv already has volume in it, so I don't think we need to have volume separately in the name.